packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

```diff
@@ -62,14 +64,50 @@ export var arrowFn = (): string => 'test';
 export var arrowFn = (arg: string): string => `test ${arg}`;
 export var arrowFn = (arg: unknown): string => `test ${arg}`;
 
-// Class is not exported
-class Test {
-  method() {
+// A class method with no return value (void)
+export class Test {
+  method(): void {
     return;
   }
 }
 ```
 
+### 2. Non-exported members may or may not have explicit signatures
```

Comment:
Formatting is a little funky here:

![Screenshot of "2. Non-exported ..." heading interpreted as a middle tab amongst five](https://github.com/typescript-eslint/typescript-eslint/assets/3335181/116b3681-491b-4792-9a43-4fba8c8226f0)

Reply:
thank you 👍
The formatting has been adjusted